src/tests/test-responses.ts

```diff
@@ -231,5 +239,11 @@ export function getRequestHandlers(): RequestHandler[] {
     );
   });
 
+  requestHandlers.push(
```

Comment:
I want to refactor the tests so they are better split instead of having this monster config... Originally I did it like this to avoid repetition, but we should be refactoring this soon